client/fakes/softlayer_client_fake.go

```diff
@@ -191,7 +209,7 @@ func (fslc *FakeSoftLayerClient) DoRawHttpRequestWithObjectMask(path string, mas
 		return []byte{}, fslc.DoRawHttpRequestError
 	}
 
-	if fslc.DoRawHttpRequestResponse != nil && len(fslc.DoRawHttpRequestResponses) == 0 {
+	if fslc.DoRawHttpRequestResponse != nil {
```

Comment:
We have added this back in, this was done for testing and was not added back in, apologies.